index.html

```diff
@@ -204,20 +220,24 @@ <h1 class="card-title">Profesional Art Printing Data</h1>
     </section>
     <section class="about">
       <div class="about-div">
-        <h1 class="about-heading">About me</h1>
-        <p class="about-descrp">
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestiate to
-          contact me.
-        </p>
-        <button class="about-btn">Get My Resume</button>
+        <div class="contents-about">
+          <h1 class="about-heading">About me</h1>
+          <p class="about-descrp">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
```

Comment:
- [ ] [REQUIRED] Kindly ensure you need to stick to the design as much as possible (font). The font type you used for the `p` tag is different from the font used in the Figma.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-06-09 13-13-56](https://user-images.githubusercontent.com/58692590/172844418-e2bda4ed-f1e3-4748-b1bd-8dd5c14fd64b.png) | ![Screenshot from 2022-06-09 13-15-07](https://user-images.githubusercontent.com/58692590/172844564-36e271ab-150e-4935-912c-8fb4da3f5d70.png)